src/openrct2-ui/WindowManager.cpp

```diff
@@ -77,8 +77,10 @@ class WindowManager final : public IWindowManager
                 return window_map_open();
             case WC_MAPGEN:
                 return window_mapgen_open();
+#ifndef DISABLE_NETWORK
```

Comment:
Can we move all of the network windows next to each other so that we don't have to have multiple ifdefs